examples/join-tables-and-streams/join_tables_and_streams.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+import ballerina/runtime;
+
+//This is the record that holds item details in the stockTable.
+type Item record {
+    string name;
+    float price;
+    int stockAmount;
+};
+
+//This is the record that holds order events from customer.
+type Order record {
+    string itemName;
+    int orderingAmount;
+};
+
+//This is the record that holds alert events.
+type OutOfStockAlert record {
+    string itemName;
+    int stockAmount;
+};
+
+// This is the input stream that uses `Order` as the constraint type.
+stream<Order> orderStream = new;
+
+// This is the table that holds the item stock data.
+table<Item> itemStockTable = table {
+    { name, price, stockAmount },
+    [
+        {"Book", 100.0, 10},
+        {"Pen", 20.0, 4}
+    ]
+};
+
+// This is the output stream that contains the events/alerts that are generated based on streaming logic.
+stream<OutOfStockAlert> oredrAlertStream = new;
+
+function initOutOfStockAlert() {
+    //whenever an order event is published to `orderStream` it is matched against the `itemStockTable` through
+    //the `queryItemTable` function. If there is a match then an alert event is published to `oredrAlertStream`.
```

Comment:
This should be a valid sentence. So start with capital letter

Reply:
done